docs/synthetics/ssl-certs.md

```diff
@@ -0,0 +1,62 @@
+
+For every run of the monitor, Synthetics performs a set of validity checks on SSL certificates sent by the server. Synthetics will mark the monitor status as failing if any of these checks fail. Synthetics will raise a run failure alert with the details of the failure. Self-signed certificates are not supported. Any websites or APIs using self-signed certificates will fail.
+
+The HTTP monitor performs the following checks:
+
+* If there is a valid chain, connecting the leaf certificate of the server to a trusted root
+* Certificate has not expired
+* Certificate is active (if the certificate is not used before its validity starts)
+* Verify name constraints, if any
+* Verify if there are too many intermediate certificates (more than 3 intermediates)
+* Hostname validation, check if the hostname of the website/API is present in the list of allowed DNS names of the certificate
+
+The browser monitor loads the website in a real Google Chrome browser so validates all the checks done by Google Chrome browser. We update the Chrome browser periodically, so the list of checks stays up-to-date with recent implementation. Along with all the above checks done by the HTTP monitor, the browser monitor performs the following extra checks:
+
+* Certificate is revoked.
+* Certificate uses a weak signature algorithm (e.g. SHA-1)
+* Certificate authority is not trusted
+* Certificate validity is too long
+* Certificate uses a weak key ( e.g. a too small RSA key)
+
+## Certificate Expiry
+
+Sematext Synthetics checks the certificate expiry every day and alerts you via the configured monitor alert channels multiple times before the expiry so that you remained about the expiry. The monitor performs the expiry check for all the certificates in the chains - leaf, intermediates, and root certificate. The Monitor Overview page displays the expiry time of the recently expiring certificate.
+
+![Certificate expiry](../images/synthetics/cert-expiry.png)
+
+## Certificate Change
+
+In the modern dynamic application environments, the SSL certificates are managed by a certificate manager, which can update the certificates automatically. This could sometimes cause issues like missing hostname in certificates or if the client time is not correct, certificate validation will fail. Sematext Synthetics checks for the certificate change, every 10 mins and notifies you via the configured monitor alert channels about the change. The monitor detects the change based on the fingerprint of the certificate. The change alert contains a detailed change report.
+
+**Certificate Change Alert**
+![Certificate change alert](../images/synthetics/cert-change-alert.png)
+
+**Certificate Change Report**
+![Certificate change report](../images/synthetics/cert-change-report.png)
+
+## SSL Certificate Report
+
+SSL certificate report provides an overview of the certificate along with the details of the intermediate and root certificates. It contains the following details:
```

Comment:
SSL certificate report provides an overview of the certificate along with the details of the intermediate and root certificates. It contains the following details:

=>

The SSL certificate report provides an overview of the certificate along with the details of the intermediate and root certificates. It contains the following details: